draft-westerlund-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -134,13 +133,12 @@ normative:
 
 * large user messages.
 
-   The method described in this document requires that the SCTP
-   implementation supports the optional feature of fragmentation of
-   SCTP user messages as defined in {{RFC4960}}. To efficiently
-   implement and support larger user messages it is also recommended
-   that I-DATA chunks as defined in {{RFC8260}} as well as an SCTP API
-   that supports partial user message delivery as discussed in
-   {{RFC6458}}.
+The method described in this document requires that the SCTP
+implementation supports the optional feature of fragmentation of SCTP
+user messages as defined in {{RFC4960}}. To efficiently implement and
+support larger user messages it is also recommended that I-DATA chunks
+as defined in {{RFC8260}} as well mandating an SCTP API that supports
+partial user message delivery as discussed in {{RFC6458}}.
```

Comment:
What about:
```
as well mandating an SCTP API (for example the one described in {{RFC6458}})
that supports partial user message delivery.
```
since even the API described in RFC 4960 does support it.

Reply:
Yes, that improves. Implemented.

Reply:
I ended up rewriting the whole sentence.